cmake/NSIS.template.in

```diff
@@ -555,6 +555,14 @@ Section /o "Open OBJ files with Cura"
     WriteRegStr HKCR "Cura OBJ model file\shell\open\command" "" '"$INSTDIR\Cura.exe" "%1"'
 SectionEnd
 
+Section /o "Open 3MF files with Cura"
+    WriteRegStr HKCR .3mf "" ""
```

Comment:
If you have a Windows machine, you could simply test it by manually adding the registry keys manually. In any case, I do not think it is a different issue. You add the file association, you should make sure that file association is more descriptive than an empty string.